### PR TITLE
Changing vagho.txt in a new branch

### PR DESCRIPTION
**Pull request by paparshakuni**
Changed the contents of vagho.txt as requested in VZ-42421.

### CODE REVIEW

File: vagho.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-This is a new file
+This is a new vagho file
```
Comment (rubentsirunyan) [inline vagho.txt]: Vagho should be upper case

**Review by rubentsirunyan — CHANGES_REQUESTED** [review]
See my coment

File: vagho.txt (hunk 1 above)
Comment (paparshakuni) [reply vagho.txt]: Done.

**Review by rubentsirunyan — APPROVED** [review]
Tuyn

**Pull request merged**